src/main/java/core/org/akaza/openclinica/service/DataImportService.java

```diff
@@ -333,8 +334,10 @@ public HashMap validateData(ODMContainer odmContainer, DataSource dataSource, Co
 
                         if (eventCRFStatus != null && eventCRFStatus.equals(DataEntryStage.INITIAL_DATA_ENTRY.getName())
                                 && eventCrfBean.getStatus().isAvailable()) {
+                        	eventCrfBean.setWorkflowStatus(EventCrfWorkflowStatusEnum.INITIAL_DATA_ENTRY);
                             crfBusinessLogicHelper.markCRFStarted(eventCrfBean, userBean, true);
                         } else {
+                        	eventCrfBean.setWorkflowStatus(EventCrfWorkflowStatusEnum.COMPLETED);
```

Comment:
shouldn't this be handled by the markCRFComplete method?

Reply:
done -- move to markCRFComplete method